spec/fortnox/api/mappers/unit_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+require 'fortnox/api'
+require 'fortnox/api/mappers/unit'
+require 'fortnox/api/models/unit'
+require 'fortnox/api/mappers/examples/mapper'
+
+module Fortnox
+  module API
+    # Shhh Rubocop, we don't need a comment here ... Really
+    module Mapper
+      describe Unit do
+        describe 'when mapping model' do
```

Comment:
I would use `context` here instead of `describe`. The convention is to use `describe` for describing methods, classes and such and use `context` to give a context to the test.